dp/generate_climos.py

```diff
@@ -516,6 +530,100 @@ def update_metadata_and_time_var(input_file, t_start, t_end, operation, climo_fi
         if hasattr(input_file, 'tracking_id'):
             cf.climo_tracking_id = input_file.tracking_id
 
+        def compare_times(t1, t2):
+            """This function takes two string inputs which are time representations
+            and returns boolean value. The purpose of this function is to find the
+            place where the start of generate_climos should be put in the history log.
+            It returns True if first input is later than the second input. Otherwise,
+            it returns False. If the second input is not in the format desired, the
+            function automatically returns True.
+            """
+            months = {
```

Comment:
How about [`calendar.month_abbr](https://docs.python.org/3/library/calendar.html#calendar.month_abbr) from the Python standard library?